BWA/0.7.17/BWASW.nf

```diff
@@ -6,13 +6,13 @@ process BWASW {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, rg_id, file(fastq: "*")
```

Comment:
I believe path(fastq:"*") should be path(fastq) , I've tried fastq:("*") and it didn't work

Reply:
Fixed :)